test/types/community/client.test-d.ts

```diff
@@ -55,6 +55,7 @@ export async function testFunc(): Promise<MongoClient> {
 
 MongoClient.connect(connectionString, err => {
   if (err instanceof MongoError) {
+    //@ts-expect-error // TODO? can we break?
```

Comment:
context for this?

Reply:
With the error label enum, typescript now can enforce you pass in one of the known labels, so narrower than string. This is a public API (although doubtful its useful outside of a driver) but this kind of type check seems like something desirable

Reply:
This causes other headaches, lets stick with string